Добавлена пересылка сообщения на которое отвечает бот

kykaroo opened the pull request.

Argon42 commented on src/DrugBot/Bot/Vk/VkBot.cs:
```diff
@@ -74,13 +74,18 @@ public void SendMessage(IVkMessage message)
             throw new InvalidOperationException("Bot not working, messages unavailable");
 
-        bool needForward = false;
+        bool needForward = true;
```
Нужно добавить получение этого параметра из конфигураций. 
IConfiguration можно получить из конструктора

Argon42 replied:
Версия немного обновилась, теперь этот параметр можно добавить в _config 
src/DrugBot.Vk/Bot/VkBotConfiguration.cs

Argon42 replied:
по дефолту должно быть выключено

kykaroo committed:
Добавлена пересылка сообщения на которое отвечает бот

-Параметр отвечающий за необходимость ответа на сообщения вынесен в фаил конфигурации (по умолчанию выключено)
